CMakeLists.txt

```diff
@@ -128,16 +139,23 @@ execute_process(
 )
 string(ASCII 27 Esc)
 if(OSQUERY_DEPS_CHECK)
-  message(WARNING "${Esc}[31m${OSQUERY_DEPS_MESSAGE}${Esc}[m")
+  message("-- ${Esc}[31m${OSQUERY_DEPS_MESSAGE}${Esc}[m")
 endif()
 
-# Generate version from git
-execute_process(
-  COMMAND git describe --tags HEAD --always
-  WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-  OUTPUT_VARIABLE OSQUERY_BUILD_VERSION
-  OUTPUT_STRIP_TRAILING_WHITESPACE
-)
+# Discover build version from an environment variable or from the git checkout.
+if(DEFINED ENV{OSQUERY_BUILD_VERSION})
```

Comment:
@zi0r let's roll with a `OSQUERY_BUILD_VERSION=1.4.5 gmake` instead of appending to CMAKE_ARGS, an environment variable is easy and may be appropriate in "other" build systems.